File: fileKu/tugas.cpp
#include <iostream>
#include <array>
using namespace std;

int main(){
    int num;
    cout << "Berapa jumlah data : "; cin >> num;
    // array<int, num>nilai;
    int nilai[num];

    cout << "\nINPUT" << endl;
    for(int i=1; i <= num; i++){
        cout << "Nilai data ke-" << i << " : ";
        cin >> nilai[i];
    }

    cout << "\nOU6TPUT" << endl;
    for(int i=1; i <= num; i++){
        cout << "Nilai data ke-" << i << " : " << nilai[i] << endl;
    }

    return 0;
}

File: fileKu/tugas.py
num = int(input("Berapa jumlah data : "))
data = []
print("\nINPUT")
for i in range(1, 1+num):
    data.insert(i, int(input(f"nilai data ke-{i} : ")))

print("\nOUTPUT")
for k, j in enumerate(data):
    print(f"nilai data ke-{k+1} : {j}")
